packages/web3/src/index.ts

```diff
@@ -29,7 +30,7 @@ import {
 import { Address } from 'web3-utils';
 import { ContractError } from './errors';
 
-export class Web3 extends Web3Context<EthExecutionAPI> {
+export class Web3 extends Web3Context<EthExecutionAPI | Web3NetAPI> {
```

Comment:
We should not change the API here. It should just be used in `web3-eth-net` package. 

```suggestion
export class Web3 extends Web3Context<EthExecutionAPI> {
```

Reply:
Understandable 👍  sounds good